index.html

```diff
@@ -51,8 +51,9 @@
         </div>
         <section class="text-container">
           <div class="intro">
-            <p>I'm Jim</p>
-            <p>Glad to see you!</p>
+            I'm Jim
+            <br />
+            Glad to see you!
```

Comment:
- [ ] [OPTIONAL] It would be nice if you add more padding in that part to center a little bit your content.
- [ ] [OPTIONAL] Add more space among social media links due to some screen sizes your social media links are so close.

![image](https://user-images.githubusercontent.com/60164239/134586992-23ec3563-f817-4c9f-9ff5-cc9c30405bdd.png)

Reply:
Thanks for the feedback.. I've been able to correct the stuff. Regards,